.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Bug report
+about: Create a report to help us reproduce and correct the bug
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+Before submitting a bug, please make sure the issue hasn't been already
+addressed by searching through the past issues.
+
+#### Describe the bug
+<!--
+A clear and concise description of what the bug is.
+-->
+
+#### Steps/Code to Reproduce
+<!--
+Please add a minimal example that we can reproduce the error by running the
+code. Be as succint as possible, do not depend on external data. In short, we
+are going to copy-paste your code and we expect to get the same
+result as you.
+
+Example:
+```python
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.decomposition import LatentDirichletAllocation
+docs = ["Help I have a bug" for i in range(1000)]
+vectorizer = CountVectorizer(input=docs, analyzer='word')
+lda_features = vectorizer.fit_transform(docs)
+lda_model = LatentDirichletAllocation(
+    n_topics=10,
+    learning_method='online',
+    evaluate_every=10,
+    n_jobs=4,
+)
+model = lda_model.fit(lda_features)
```

Comment:
This example could be simplified to use `np.ranom.RandomState(42).normal(size=(n_samples, n_features))` and a more common estimator such as `RandomForestClassifier` instead.  But let's do that in a later PR.